src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -12,18 +52,18 @@ fun scenario(vehicleList: List<Vehicle>, road: RoadNetwork){
 
 val testRoad: RoadNetwork = RoadNetwork(8.0)
 
-// Creation of sufficient cars for local testing without using CSV input for now
```

Comment:
Please keep the scenario that is created in code, so it can be executed even if there are problems with CSV writing/parsing. (I didn't mention replacing it and future tasks actually extend it.)

(Mist, ich kann einfach englischen Code nicht auf Deutsch kommentieren...)